Config/Routes.js

```diff
@@ -1,10 +1,20 @@
-const ProductController = require("../Controllers/ProductController");
+const DbService = require('../Services/DbService')
+const ProductController= require("../Controllers/ProductController");
 
 function routes(app) {
     app.get('/products', ProductController.getAllProducts)
     app.post('/products', ProductController.addSingleProduct)
+    app.put('/products', ProductController.updateSingleProduct)
```

Comment:
should this not also be invalid as the 'updateSingleProduct' requires the 'SKU' in the URL and has a route on line 11